kani-compiler/src/kani_middle/transform/body.rs

```diff
@@ -288,3 +284,136 @@ impl SourceInstruction {
 fn find_instance(tcx: TyCtxt, diagnostic: &str) -> Option<Instance> {
     Instance::resolve(find_fn_def(tcx, diagnostic)?, &GenericArgs(vec![])).ok()
 }
+
+/// Basic mutable body visitor.
+///
+/// We removed a many methods for simplicity.
+///
+/// TODO: Contribute this to stable_mir.
```

Comment:
Should we track this in a GitHub issue?

Reply:
Sure: https://github.com/rust-lang/project-stable-mir/issues/81